### PR TITLE
xxlimited_35 module now has the same name in repr in Py 3.11

### PR DESCRIPTION
**Pull request by frenzymadness**
See https://github.com/python/cpython/commit/a87c9b538fbfc42883417c4d5e69f1a5922690e3

### CODE REVIEW

**Comment by frenzymadness** [comment]
This change has been introduced in Python 3.11 beta 4.

**Comment by bnavigator** [comment]
Also Python 3.10.6: https://github.com/python/cpython/pull/93679

File: IPython/lib/tests/test_pretty.py [hunk 1]
```diff
@@ -141,9 +141,12 @@ def test_pprint_heap_allocated_type():
     Test that pprint works for heap allocated types.
     """
     module_name = "xxlimited" if sys.version_info < (3, 10) else "xxlimited_35"
+    expected_output = (
+        "xxlimited.Null" if sys.version_info < (3, 11) else "xxlimited_35.Null"
```
Comment (bnavigator) [inline IPython/lib/tests/test_pretty.py:145]:
```suggestion
        "xxlimited.Null" if sys.version_info < (3, 10, 6) else "xxlimited_35.Null"
```

Comment (frenzymadness) [reply IPython/lib/tests/test_pretty.py]: Good catch, thank you!

**Comment by Carreau** [comment]
thanks

**Comment by Carreau** [comment]
@meeseeksdev backport to 7.x

**Pull request merged**